### ISSUE TITLE
make the two ways of using the faucet more explicit

### ISSUE THREAD

**Issue by nicolasochem** [avysel/tezos-faucet#11]
* state in plain english: "Fund your Web Wallet" "Fund Any Address"
* replace the "tz" symbol with "tokens" in the buttons. one tez is implicitely "one mainnet tez". testnet tokens should be referred to as "tokens"

See attached picture

![PXL_20220818_235703166](https://user-images.githubusercontent.com/93067/185514957-5034aa35-08d4-4b1d-a431-f7d9a8c21db8.jpg)

**Comment by avysel** [comment]
https://github.com/avysel/tezos-faucet/pull/12
